src/ngraph/builder/reshape.cpp

```diff
@@ -258,58 +258,34 @@ namespace ngraph
 
 shared_ptr<Node> builder::opset1::flatten(const Output<Node>& value, int axis)
 {
-    if (value.get_partial_shape().is_static())
+    shared_ptr<Node> output_shape;
+    if (axis == 0)
     {
-        auto data_shape = value.get_shape();
-        // First dimension of output tensor is the product of [d_0, ... d_{axis-1}] dimensions of
-        // input
-        // tensor. The last dimension is the product of the rest of input tensor dimensions:
-        // [d_{axis}, ..., d_n]
```

Comment:
I'd leave this comment inline. It's helpful at least for me, since I always forget whether first part is inclusive/exclusive the split axis.

Reply:
good point, done